Modules/Devices/test/Devices.Domain.Tests/Identities/CancelDeletionProcessTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Backbone.BuildingBlocks.Domain;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+using Backbone.Tooling;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Devices.Domain.Tests.Identities;
+public class CancelDeletionProcessTests
+{
+    [Fact]
+    public void Cancel_deletion_process()
+    {
+        // Arrange
+        var currentDate = DateTime.Parse("2020-01-01");
+        SystemTime.Set(currentDate);
```

Comment:
You don't even assert on the `SystemTime` you set here. So why do you even set it?